package.json

```diff
@@ -903,6 +904,11 @@
                 "title": "%mssql.schemaCompare%",
                 "category": "MS SQL"
             },
+            {
```

Comment:
This file can be reverted because we're now updating an existing command instead of adding a new one.